src/docs/content/builders/dialog.md

```diff
@@ -20,7 +20,18 @@ At a high level, the anatomy of a dialog looks like this:
 ```svelte
 <script lang="ts">
 	import { createDialog, melt } from '@melt-ui/svelte'
-	const { trigger, portal, overlay, content, title, description, close, open } = createDialog()
+	const { 
+		elements: {
+			trigger, 
+			portal, 
```

Comment:
It should be `portalled`, instead of portal. Also, the docs are missing the API reference for `portalled`, could you add it in?

Reply:
sure, can you explain what exactly i have to add and where